spec/services/end_of_cycle_timetable_spec.rb

```diff
@@ -7,137 +7,223 @@
       end
-    end
 
-    it 'returns true between find_closes and find_reopens' do
-      Timecop.travel(EndOfCycleTimetable.find_closes.end_of_day + 1.hour) do
-        expect(EndOfCycleTimetable.find_down?).to be true
+      it 'returns true between find_closes and find_reopens' do
+        Timecop.travel(EndOfCycleTimetable.find_closes.end_of_day + 1.hour) do
+          expect(EndOfCycleTimetable.find_down?).to be true
+        end
       end
     end
-  end
 
-  describe '.current_cycle?' do
-    def create_application_for(recruitment_cycle_year)
-      create :application_form, recruitment_cycle_year: recruitment_cycle_year
-    end
+    describe '.current_cycle?' do
+      def create_application_for(recruitment_cycle_year)
+        create :application_form, recruitment_cycle_year: recruitment_cycle_year
+      end
 
-    it 'returns true for an application for courses in the current cycle' do
-      expect(
-        described_class.current_cycle?(create_application_for(RecruitmentCycle.current_year)),
-      ).to be true
+      it 'returns true for an application for courses in the current cycle' do
+        expect(
+          described_class.current_cycle?(create_application_for(RecruitmentCycle.current_year)),
+        ).to be true
+      end
+
+      it 'returns false for an application for courses in the previous cycle' do
+        expect(
+          described_class.current_cycle?(create_application_for(RecruitmentCycle.previous_year)),
+        ).to be false
+      end
     end
 
-    it 'returns false for an application for courses in the previous cycle' do
-      expect(
-        described_class.current_cycle?(create_application_for(RecruitmentCycle.previous_year)),
-      ).to be false
+    describe 'stop_applications_to_unavailable_course_options?' do
+      it 'is true when between "stop_applications_to_unavailable_course_options" and "apply_reopens"' do
+        Timecop.travel(Time.zone.local(2020, 9, 7).end_of_day + 1.minute) do
+          expect(EndOfCycleTimetable.stop_applications_to_unavailable_course_options?).to be true
+        end
+      end
+
+      it 'is false when before the window' do
+        Timecop.travel(Time.zone.local(2020, 9, 7).end_of_day - 1.minute) do
+          expect(EndOfCycleTimetable.stop_applications_to_unavailable_course_options?).to be false
+        end
+      end
+
+      it 'is false when after the window' do
+        Timecop.travel(Time.zone.local(2020, 10, 13).beginning_of_day) do
+          expect(EndOfCycleTimetable.stop_applications_to_unavailable_course_options?).to be false
+        end
+      end
     end
   end
 
-  describe 'stop_applications_to_unavailable_course_options?' do
-    it 'is true when between "stop_applications_to_unavailable_course_options" and "apply_reopens"' do
-      Timecop.travel(Time.zone.local(2020, 9, 7).end_of_day + 1.minute) do
-        expect(EndOfCycleTimetable.stop_applications_to_unavailable_course_options?).to be true
+  context 'end of cycle policy' do
```

Comment:
We've added this context to make it clear what was moved over from the old EndOfCyclePolicy class but it's possible this may not be required following future refactoring as per the _plan_

Reply:
I think we could do without this — it's a vestige of the refactor, and we'll just remove in next PR anyway?